internal/controller/data_plane_policies_workflow.go

```diff
@@ -119,3 +127,24 @@ func gatewayComponentsToSync(gateway *machinery.Gateway, componentGroupKind sche
 	}
 	return nil
 }
+
+func getGatewayControllerNames(envName string, defaultGatewayControllerName string) []gatewayapiv1.GatewayController {
+	envValue := env.GetString(envName, defaultGatewayControllerName)
+	gatewayControllers := utils.Map(strings.Split(envValue, ","), func(c string) gatewayapiv1.GatewayController {
```

Comment:
Can we please use `github.com/samber/lo` here and avoid importing another package?

Reply:
is there any equivalent to `strings.Split` in `lo` ?

Reply:
There is nothing wrong with `utils.Map`. As long as it is there, we are free to use it. 

@guicassolato if you think `github.com/samber/lo` is a better library than 4 lines of code on our own repo, be my guest and open a PR deleting the `utils.Map` function from the repo.

Reply:
I think he means to avoid including `strings` and `slices` and use the already imported `github.com/samber/lo`

Reply:
`strings` and `slices` packages come from the standard library, which is always more stable and performant than any other third party lib. 

@guicassolato can you clarify your request?

Reply:
Maybe we could decide if it makes sense to remove the slice utils in favor of `lo` (not as part of this PR tho)